notebooks/template/template.ipynb

```diff
@@ -2,41 +2,53 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "id": "bee45446-653f-4074-a0af-27b3e19b2f1f",
-   "metadata": {
-    "tags": []
-   },
+   "id": "18cf7739-a9b9-496c-973d-c36f8f3e0515",
+   "metadata": {},
```

Comment:
I noticed that by mistake I removed the metadata part in this commit https://github.com/EOPF-Sample-Service/eopf-sample-notebooks/commit/39a4051a95da92b7b60d341053df1dee1248b192 . Should we still have it or not?

Reply:
I added them manually. I will investigate about that.